### PR TITLE
LPS-170508 - Notification templates with the same ERC don't show error message

### PR DESCRIPTION
**Pull request by Artartur**
[LPS-170508](https://issues.liferay.com/browse/LPS-170508)

### CODE REVIEW

**Comment by Artartur** [comment]
ci:test:sf

**Comment by Artartur** [comment]
ci:test:relevant

**Comment by Artartur** [comment]
ci:test:relevant

**Comment by Artartur** [comment]
ci:test:relevant

**Comment by Artartur** [comment]
ci:test:relevant

**Comment by Artartur** [comment]
ci:test:relevant

**Comment by Artartur** [comment]
ci:test:relevant

File: modules/apps/notification/notification-rest-impl/src/main/java/com/liferay/notification/rest/internal/jaxrs/exception/mapper/DuplicateListTypeDefinitionExternalReferenceCodeExceptionMapper.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+ */
+@Component(
+	property = {
+		"osgi.jaxrs.application.select=(osgi.jaxrs.name=Liferay.Notification.REST)",
+		"osgi.jaxrs.extension=true",
+		"osgi.jaxrs.name=Liferay.Notification.REST.DuplicateNotificationTemplateExternalReferenceCodeExceptionMapper"
+	},
+	service = ExceptionMapper.class
+)
+public class DuplicateListTypeDefinitionExternalReferenceCodeExceptionMapper
```
Comment (murilostodolni) [inline modules/apps/notification/notification-rest-impl/src/main/java/com/liferay/notification/rest/internal/jaxrs/exception/mapper/DuplicateListTypeDefinitionExternalReferenceCodeExceptionMapper.java:37]: Hi @Artartur great job here!

I need that you make a little change:
Please, rename class to `DuplicateNotificationTemplateExternalReferenceCodeExceptionMapper`

**Review by murilostodolni — CHANGES_REQUESTED** [review]

**Comment by murilostodolni** [comment]
ci:stop

File: modules/apps/notification/notification-rest-impl/src/main/java/com/liferay/notification/rest/internal/jaxrs/exception/mapper/DuplicateListTypeDefinitionExternalReferenceCodeExceptionMapper.java (hunk 1 above)
Comment (Artartur) [reply modules/apps/notification/notification-rest-impl/src/main/java/com/liferay/notification/rest/internal/jaxrs/exception/mapper/DuplicateListTypeDefinitionExternalReferenceCodeExceptionMapper.java]: Done!

**Comment by naraandrade** [comment]
ci:test:sf

**Comment by naraandrade** [comment]
Just started reviewing :)

**Review by murilostodolni — APPROVED** [review]
LGTM 🎉

**Comment by murilostodolni** [comment]
Waiting for @naraandrade 's review on the front-end.

**Review by naraandrade — CHANGES_REQUESTED** [review]
@Artartur you don't need to make any changes in the front side. I tested with only the back-end changes and all works well. Please remove your changes on `ModalEditExternalReferenceCode`. Thanks :)

**Comment by Artartur** [comment]
ci:test:sf

**Comment by naraandrade** [comment]
Just started reviewing :)

**Review by naraandrade — APPROVED** [review]

**Comment by naraandrade** [comment]
ci:forward

**Pull request closed without merging**